iosApp/iosApp/Pages/Search/SearchResultView.swift

```diff
@@ -75,15 +59,11 @@ struct SearchView: View {
     }
 
     var body: some View {
-        VStack {
-            if !query.isEmpty {
-                SearchResultView(
-                    results: searchResults,
-                    handleStopTap: handleStopTap,
-                    showRoutes: searchVM.routeResultsEnabled
-                )
-            }
-        }
+        SearchResultView(
```

Comment:
question (non-blocking): Is there an accessibility label for this section that would be helpful to introduce it as search results?